plugins/mcreator-core/aitasks/$othertasks.json

```diff
@@ -1,3 +1,3 @@
 {
-  "color": 250
+  "color": 300
```

Comment:
Blocks in other tasks should be set to 300 too as right now they have same color as advanced blocks which is not ok